From 6685400a6434db4274d8ce5ffe21313a1f98ced4 Mon Sep 17 00:00:00 2001
From: Fabrizio Castro <fabrizio.castro.jz@renesas.com>
Date: Wed, 24 Nov 2021 21:38:39 +0000
Subject: [PATCH] arm64: dts: renesas: r9a07g044l: Add missing companion chip
 references

The EHCI nodes are missing the phandles for the companion IPs, add them.

Signed-off-by: Fabrizio Castro <fabrizio.castro.jz@renesas.com>
---
 arch/arm64/boot/dts/renesas/r9a07g044l.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi b/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
index c382166f2c15..8495d7919944 100644
--- a/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
+++ b/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
@@ -431,6 +431,7 @@
 			resets = <&cpg R9A07G044L_CLK_USB0>;
 			phys = <&usbtest>, <&usb2_phy0>;
 			phy-names = "usb";
+			companion = <&ohci0>;
 			power-domains = <&cpg>;
 			status = "disabled";
 		};
@@ -455,6 +456,7 @@
 			resets = <&cpg R9A07G044L_CLK_USB1>;
 			phys = <&usbtest>, <&usb2_phy1>;
 			phy-names = "usb";
+			companion = <&ohci1>;
 			power-domains = <&cpg>;
 			status = "disabled";
 		};
-- 
2.25.1

